modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-raid_{context}"]
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume. {product-title} supports RAID 0, RAID 1, RAID 4, RAID 5, RAID 6, and RAID 10 for data protection and fault tolerance. See "About disk mirroring" for more details.
```

Comment:
Does RAID need to be spelled out on first reference?

Reply:
I tried spelling it out but it needs to say "software RAID" and spelling it out as "software redundant array of independent disks (RAID)" made it too hard to understand, imo. I found 1 or 2 other instances in our docs where we also don't spell it out, so I'm inclined to leave it this way.